src/vos/vos_layout.h

```diff
@@ -343,8 +347,10 @@ struct vos_obj_df {
 	daos_unit_oid_t			vo_id;
 	/** The latest sync epoch */
 	daos_epoch_t			vo_sync;
-	/** Attributes of object.  See vos_oi_attr */
-	uint64_t			vo_oi_attr;
+	/** Offset of known existing dkey */
+	umem_off_t			vo_known_dkey;
+	/** Attributes for future use */
+	uint64_t			vo_attr;
```

Comment:
What is the purpose of keep such useless "attr" field?

Reply:
@gnailzenh asked me to add it back for future use